packages/aws-cdk-api/package.json

```diff
@@ -0,0 +1,144 @@
+{
+  "name": "aws-cdk-api",
+  "description": "AWS CDK Programmatic API",
+  "version": "0.0.0",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "java": {
+        "package": "software.amazon.awscdk.api",
```

Comment:
That's a bold name! Not sure how I feel about it 😬

Reply:
Agreed. The linter forced that name based on the package name. But we might change the package name.